scripts/setup

```diff
@@ -370,8 +370,10 @@ then
     eval ${PREFIX} invenio oaiharvester harvest -n ebooks -q -k
```

Comment:
Should not be mandatory if we make a deployment, but that ebooks are not needed.